home/src/main/resources/config/example.runtime.properties

```diff
@@ -92,6 +92,14 @@ email.replyTo =
 #
 # listview.usePreciseSubquery = true
 
+#
+# The Map of Science visualization is limited in the number of supported journals.
+# This will generally only be useful for institutions that specifically want this.
+# This is enabled by default.
+# Set this to either "disabled" to not utilize or "enabled" to utilize.
+#
+visualization.mapOfScience = enabled
+
```

Comment:
I think mapOfScience can't work in Vitro, therefore not sure there is a sense in this configuration in example.runtime.properties in Vitro project, I think we need that only in VIVO project (similar as visualization. temporal which I believe is only defined in VIVO runtime.properties). Later, in a separate issue, I hope we might even move the code from Vitro IndividualResponseBuilder to the VIVO codebase.